DOCS-2789: QA capture sync image data how-to with filtered camera (#3295)
viamrobotics/docs: docs/how-tos/image-data.md

In the **Data capture** panel of your camera's configuration, select **Read image** from the method selector.

Set your desired capture frequency.
For example, set it to `0.05` to capture an image every 20 seconds.

Set the MIME type to your desired image format, for example `image/jpeg`.

Click **Save** in the top right corner of the screen to start capturing data.

{{% /tablestep %}}
{{% tablestep %}}
{{<imgproc src="/services/ml/collect.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
**4. View data in the Viam app**

Click on the **...** menu of the camera component and click on **View captured data**.
This takes you to the data tab.

![View captured data option in the component menu](/get-started/quickstarts/collect-data/cam-capt-data.png)

If you do not see images from your camera, try waiting a minute and refreshing the page to allow time for the images to be captured and then synced to the app at the interval you configured.

If no data appears after the sync interval, check the [**Logs**](/cloud/machines/#logs).

> You can also [export your data from the Viam app](/services/data/export/) to a deployed machine, or to any computer.

Reviewer: This is very random here
```suggestion
```

You now know how to capture and sync image data.

{{< alert title="Avoid fees by disabling data capture" color="note" >}}

> If this is not a production machine and you have confirmed that your machine is capturing and syncing data, you should now disable data capture in the camera configuration to avoid [incurring fees](https://www.viam.com/product/pricing) for capturing large amounts of test data.

Reviewer: it might be good to make that an explicit step like on the sensor how-to. That make it less easy to miss it I think

{{<imgproc src="/services/ml/train.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
**2. Select a suitable ML model**

Click **Select model** on the ML model service configuration panel, then select an [existing model](https://app.viam.com/registry?type=ML+Model) you want to use, or click **Add new model** to upload your own.

> If you're not sure which model to add, you can add [`face-detection`](https://app.viam.com/ml-model/bijan/face-detection) from the **Registry**, which can detect if a person's face appears in the image stream.

Reviewer: because you just use add as "adding a model by uploading it"
```suggestion
If you're not sure which model to add, you can use [`face-detection`](https://app.viam.com/ml-model/bijan/face-detection) from the **Registry**, which can detect if a person's face appears in the image stream.
```

The `filtered-camera` {{< glossary_tooltip term_id="modular-resource" text="modular component" >}} pulls the stream of images from the camera you configured earlier, and applies the vision service to it.

Configure a `filtered-camera` component on your machine, following the [attribute guide in the README](https://github.com/erh/filtered_camera?tab=readme-ov-file#configure-your-filtered-camera).
Use the name of the camera you configured in the first part of this guide as the `"camera"` to pull images from, and select the name of the vision service you just configured as your `"vision"` service.
Then add all or some of the labels your ML model uses as classifications or detections in `"classifications"` or `"objects"`.

> For example, if you are using the `face-detection` model, you could use a configuration like the following to only capture images when a face is detected in your camera stream:

Reviewer 2: ```suggestion
For example, if you are using the `face-detection` model, you could use a configuration like the following to only capture images when the detector detects a face in your camera stream with 80% confidence or higher:
```
[opt] so they know what the .8 means 

BTW I’m still confused by the buffer window seconds attribute—since it’s optional I never used it but do you know? If so maybe explain? Or remove if not needed?

```json {class="line-numbers linkable-line-numbers"}
{
  "window_seconds": 1,
  "objects": {
    "face": 0.8
  },
  "camera": "camera-1",
  "vision": "vision-1"
}
```

{{% /tablestep %}}
{{% tablestep %}}
{{<imgproc src="/services/icons/data-capture.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
**5. Configure data capture and sync on the filtered camera**

Configure data capture and sync on the filtered camera just as you did before for the physical camera.
The filtered camera will only capture image data that passes the filters you configured in the previous step.

Turn off data capture on your original camera if you haven't already, so that you don't capture duplicate or unfiltered images.

{{% /tablestep %}}

> {{% tablestep %}}

Reviewer: The step 6 viewing filtered data, I think possibly better to do that the same way as in step 4 above (not linking out)

Reviewer: And I think the query sensor data card at the end should probably be replaced with https://docs-test.viam.dev/3295/services/data/export/

The `filtered-camera` {{< glossary_tooltip term_id="modular-resource" text="modular component" >}} pulls the stream of images from the camera you configured earlier, and applies the vision service to it.

Configure a `filtered-camera` component on your machine, following the [attribute guide in the README](https://github.com/erh/filtered_camera?tab=readme-ov-file#configure-your-filtered-camera).
Use the name of the camera you configured in the first part of this guide as the `"camera"` to pull images from, and select the name of the vision service you just configured as your `"vision"` service.
Then add all or some of the labels your ML model uses as classifications or detections in `"classifications"` or `"objects"`.

For example, if you are using the `EfficientDet-COCO` model, you could use a configuration like the following to only capture images when a person is detected in your camera stream.
You can also add a time window to capture data in right before a detection, so as to, for example, capture the 3 seconds before a person appeared in the stream.

```json {class="line-numbers linkable-line-numbers"}
{
  "window_seconds": 0,
  "objects": {

>     "Person": 0.5

Author: ```suggestion
    "Person": 0.8
```
